go/sgf.h

```diff
@@ -49,6 +49,16 @@ inline string coord2str2(Coord c) {
     return s + std::to_string(y + 1);
 }
 
+inline string coords2sgfstr(const vector<Coord>& moves) {
```

Comment:
This is kind of dirty because I'm not creating a proper Sgf object and then writing a proper to_string function here.

Is this correct sgf syntax? I'm not sure what I'm required to include in the header.